src/utils/parsePhone.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { parsePhoneNumber } from "./parsePhone";
+
+describe("Normal number", () => {
+    it("should return a string of the normal phone number", () => {
+
+        const want = "(07) 873 7079";
+
+        const got = parsePhoneNumber("{ telephone: '(07) 873 7079  /  A/H (07) 873 8399' }");
+
```

Comment:
Reduce whitespace please, no need for blank lines in between. Tip: Also run prettier for formatting. We use an indentation of tab size: 2 throughout this project (not 4 spaces indentation) - Prettier should fix this for you.